.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,37 @@
+name: Check build pass/fail
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+      with:
+        persist-credentials: false
+
+    # Setup the flutter web build
+    - uses: subosito/flutter-action@v1.2.0
+      with:
+        channel: beta
+      if: ${{ runner.os == "Windows" }}
+      - run: flutter config --enable-windows-desktop
+      if: ${{ runner.os == "macOS" }}
+    - run: flutter config --enable-macos-desktop
+      if: ${{ runner.os == "Linux" }}
+    - run: flutter config --enable-linux-desktop
+    - run: flutter upgrade    
+    if: ${{ runner.os == "Linux" }}
+    - run: flutter config --enable-web
+    - run: flutter upgrade
+    - run: flutter build web --release
```

Comment:
You're not making a desktop binary? Also remember to upload the artifact.
And maybe put the `--enable-web` before the upgrade so you don't run it twice.